### PR TITLE
 metrics: define baselines for the new metrics CI node 

### PR DESCRIPTION
**Pull request by devimc**

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

File: cmd/container-manager/manage_ctr_mgr.sh [hunk 1]
```diff
@@ -107,7 +107,12 @@ install_docker(){
 			curl -fsSL "${repo_url}/gpg" | sudo apt-key add -
 			sudo -E add-apt-repository "deb [arch=${arch}] ${repo_url} $(lsb_release -cs) stable"
 			sudo -E apt-get update
-			docker_version_full=$(apt-cache madison $pkg_name | grep "$docker_version" | awk '{print $3}' | head -1)
+			if [ "$(echo "${VERSION_ID} <= 18.04" | bc -q)" == "1" ]; then
+				docker_version_full=$(apt-cache madison $pkg_name | grep "$docker_version" | awk '{print $3}' | head -1)
+			else
+				# Latest ubuntu may not have he version of docker that we need
```
Comment (chavafg) [inline cmd/container-manager/manage_ctr_mgr.sh:113]: typo: `s/he/the`?

**Review by chavafg — APPROVED** [review]
thanks @devimc, looks good, just a small nit.

**Pull request merged**